### PR TITLE
fixed icon issue 

### PR DESCRIPTION
**Pull request by JavedHussain-06**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
 900 - 1200px:   Tablet landscape
 [1200 - 1800] is where our normal styles apply
 1800px + :      Big desktop
-$breakpoint arguement choices:
+$breakpoint argument choices:
```
Comment (JavedHussain-06) [inline css/style.css:7]: here's a correct one

File: css/style.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
 900 - 1200px:   Tablet landscape
 [1200 - 1800] is where our normal styles apply
 1800px + :      Big desktop
-$breakpoint arguement choices:
```
Comment (JavedHussain-06) [inline css/style.css:7]: here's a typo

File: css/style.css [hunk 3]
```diff
@@ -145,12 +144,12 @@ textarea {
 }
 
 .heading-sec__main--lt::after {
-  content: '';
+  content: "";
```
Comment (JavedHussain-06) [inline css/style.css:147]: i just formatted the code

File: css/style.css [hunk 4]
```diff
@@ -750,10 +763,10 @@ textarea {
   top: 100%;
   visibility: hidden;
   opacity: 0;
-  transition: all .3s;
-  box-shadow: 0px 5px 5px 0px rgba(0, 0, 0, 0.1);
-  -webkit-box-shadow: 0px 5px 5px 0px rgba(0, 0, 0, 0.1);
+  transition: all 0.3s;
   -moz-box-shadow: 0px 5px 5px 0px rgba(0, 0, 0, 0.1);
```
Comment (JavedHussain-06) [inline css/style.css:767]: you must first define" -moz-box-shadow" , "-webkit-box-shadow"  before this "box-shadow" to work properly

File: index.html [hunk 5]
```diff
@@ -3,23 +3,29 @@
...
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css"
+    />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <meta name="description" content="Portfolio Template for Developer" />
     <link rel="stylesheet" href="css/style.css" />
 
-    <!-- Font Famliy -->
```
Comment (JavedHussain-06) [inline index.html:11]: here's a another typo

File: index.html [hunk 6]
```diff
@@ -107,7 +117,12 @@ <h1 class="heading-primary">Hey, My name is Mastoora Turkmen</h1>
       </div>
       <div class="home-hero__socials">
         <div class="home-hero__social">
-          <a target="_blank" href="https://www.linkedin.com/in/mastoora-turkmen/" class="home-hero__social-icon-link">
```
Comment (JavedHussain-06) [inline index.html:110]: i adde rel="noopener noreferrer" coz it showing warning ⚠ "noopener" helps prevent tabnabbing attacks, and "noreferrer" enhances privacy and prevents the linked website from knowing the exact URL of the page that referred the user.

File: index.html [hunk 7]
```diff
@@ -3,23 +3,29 @@
...
 
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600;700;900&display=swap"
       rel="stylesheet"
-      />
-      <title>Mastoora Turkmen Portfolio</title>
+    />
+    <script
```
Comment (JavedHussain-06) [inline index.html:22]: i added font awesome kit script to use it's icon in this project
[https://fontawesome.com/](https://fontawesome.com/) here's the link you visit explore yourself

File: index.html [hunk 8]
```diff
@@ -331,44 +376,48 @@ <h2 class="heading heading-sm main-footer__heading-sm">
...
-                  alt="icon"
-                />
+              <a
+                class="main-footer__icon"
+                target="_blank"
+                title="linkedin"
+                rel="noopener noreferrer"
+                href="https://www.linkedin.com/in/mastoora-turkmen/"
+              >
+                <i class="fa-brands fa-linkedin fa-2xl"></i>
```
Comment (JavedHussain-06) [inline index.html:377]: this how we implement a icon
[https://fontawesome.com/docs/web/setup/get-started](https://fontawesome.com/docs/web/setup/get-started) here you can check it yourself

**Review by MastooraTurkmen — COMMENTED** [review]
Thank you, I just finished reviewing.

**Pull request merged**